frontend/app/src/components/Sidebar/ThemedSidebar.test.tsx

```diff
@@ -35,36 +37,34 @@ function getProps(
 
 describe("ThemedSidebar Component", () => {
   it("should render without crashing", () => {
-    const wrapper = mount(<ThemedSidebar {...getProps()} />)
+    render(<ThemedSidebar {...getProps()} />)
 
-    expect(wrapper.find("Sidebar").exists()).toBe(true)
+    expect(screen.getByTestId("stSidebar")).toBeInTheDocument()
   })
 
   it("should switch bgColor and secondaryBgColor", () => {
-    const wrapper = mount(<ThemedSidebar {...getProps()} />)
+    render(<ThemedSidebar {...getProps()} />)
 
-    const updatedTheme = wrapper.find("Sidebar").prop("theme")
-
-    // @ts-expect-error
-    expect(updatedTheme.colors.bgColor).toBe(
-      emotionLightTheme.colors.secondaryBg
-    )
-    // @ts-expect-error
-    expect(updatedTheme.inSidebar).toBe(true)
+    const sidebar = screen.getByTestId("stSidebar")
+    expect(sidebar).toBeInTheDocument()
+    expect(sidebar).toHaveStyle({
+      backgroundColor: emotionLightTheme.colors.secondaryBg,
+    })
```

Comment:
super nit:
this can be condensed to:
```
expect(screen.getByTestId("stSidebar")).toHaveStyle({
      backgroundColor: emotionLightTheme.colors.secondaryBg,
    })
```